src/ducks/categories/CategoriesHeader.jsx

```diff
@@ -53,6 +62,38 @@ const CategoryAccountSwitch = ({ selectedCategory, breadcrumbItems }) => {
   )
 }
 
+const CategoriesTableHead = props => {
+  const { selectedCategory } = props
+  const { isDesktop, isTablet } = useBreakpoints()
+  const { t } = useI18n()
+  return (
+    <thead>
+      <tr>
+        <td className={stCategory}>
+          {t(
+            `Categories.headers.${
+              selectedCategory ? 'subcategories' : 'categories'
+            }`
+          )}
+        </td>
+        {(isDesktop || isTablet) && (
```

Comment:
I would say yes that this means the same. *But*, I am not sure we should use either isDesktop or isTablet. This is historical code that has just been moved. Having specific code for tablets is very risky as this is not a viewport we test often, and this leads to bugs. AMHA we should only have desktop and mobile code.

I plan to make a pass on Banks to see all the bugs we have due to us trying to support tablets, but this will not be solved in this PR.

cc @joel-costa @fffflo